build/buildconstants/params_calibnet.go

```diff
@@ -30,80 +30,77 @@ var ActorDebugging = false
 const BootstrappersFile = "calibnet.pi"
 const GenesisFile = "calibnet.car.zst"
 
-const UpgradeBreezeHeight = -1
+const UpgradeBreezeHeight abi.ChainEpoch = -1
```

Comment:
Are the types necessary?

Reply:
Ah, is this to help unify with `params_2k` to ensure that if this builds, that builds?

Reply:
It's not essential, it's just consistent, it doesn't get properly checked because of the build excludes. I'll back it out if you object but the inconsistency got to me after it tripped me up with one particular codesite (can't remember now, but it was do to with epoch arithmetic where it didn't know it should be a `ChainEpoch`.

Reply:
No, I've been hit by it before as well.